xbmc/windowing/X11/WinSystemX11GLESContext.cpp

```diff
@@ -1,6 +1,6 @@
 #include "system.h"
 
-#if defined(HAVE_X11) && defined(HAS_EGL)
+#if defined(HAVE_X11) && defined(HAS_GLES)
```

Comment:
No, not by accident. It doesn't build without it.
Thinking of it, it probably should be
<code>if defined(HAVE_X11) && defined(HAS_EGL) || defined(HAS_GLES)</code>